src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -153,6 +167,26 @@ def load_command_table(self, _):
         g.command('delete', 'delete')
         g.generic_update_command('update', custom_func_name='update_image')
 
+    with self.command_group('image template', image_builder_image_templates_sdk, custom_command_type=image_builder_custom, is_preview=True) as g:
```

Comment:
This is the only `is_preview` entry you need. All of the subgroups are deemed to be in preview by virtue of this. Won't block on it, but please make a follow-up PR to clean up.